doc/releases/changelog-0.37.0.md

```diff
@@ -167,35 +167,135 @@ via the `NoiseModel` class and an `add_noise` transform.
+    [pldb]: print(tape.draw(wire_order=[0,1,2]))
+    0: ──H─╭●─────┤  
+    1: ────│───RX─┤  
+    2: ────╰X──RY─┤ 
+    [pldb]: tape.operations
+    [Hadamard(wires=[0]), CNOT(wires=[0, 2])]
+    ```
+  - `qml.debug_state()` - equivalent to `qml.state()`, gives the current state.
+    ```pycon
+    [pldb]: print(qml.debug_state())
+    [0.70710678+0.j 0.        +0.j 0.        +0.j 0.        +0.j
+     1.        +0.j 0.70710678+0.j 0.        +0.j 0.        +0.j]
+    ```
+  Other debugger functions like `qml.debug_probs()` and `qml.debug_expval()` also function like their simulation counterparts (`qml.probs` and `qml.expval`, respectively) and are described in more detail in the [debugger documentation](https://docs.pennylane.ai/en/stable/code/qml_debugging.html)
+  
+  Additionally, standard debugging commands are available to navigate through code:
+  - `list` or `longlist` - view the current position and surrounding code.
+    ```pycon
+    [pldb]: longlist
+     2     @qml.qnode(qml.device('default.qubit', wires=(0,1,2)))
+     3     def circuit(x):
+     4         qml.Hadamard(wires=0)
+     5         qml.CNOT(wires=(0,2))
+     6         qml.breakpoint()
+     7  
+     8  ->     qml.RX(-x, wires=1)
+     9         qml.RY(-x, wires=2)
+    10         qml.breakpoint()
+    11  
+    12         return qml.sample()
+    ```
+  - `next` - apply the next operation in the circuit.
+    ```pycon
+    [pldb]: next
+    > /Users/your/path/to/script.py(9)circuit()
+    -> qml.RY(-x, wires=2)
+    ```
+  - `continue` - continue to execute the code until another breakpoint is reached.
+    ```pycon
+    [pldb]: continue
+    > /Users/your/path/to/script.py(12)circuit()
+    -> return qml.sample()
+    ```
+  - `quit` - exit debugging.
+  
+  Finally, to modify a circuit mid-run, simply call the desired PennyLane operations:
+
+  ```pycon
+  > /Users/your/path/to/script.py(12)circuit()
+  -> return qml.sample()
+  [pldb] qml.CNOT(wires=(0,2))
+  CNOT(wires=[0, 2])
+  [pldb]: print(qml.debug_tape().draw(wire_order=[0,1,2]))
+  0: ──H─╭●─────╭●─┤  
+  1: ────│───RX─│──┤  
+  2: ────╰X──RY─╰X─┤ 
+  ```
 
 <h4>Convert between OpenFermion and PennyLane 🤝</h4>
 
-* The ``from_openfermion`` and ``to_openfermion`` functions are added to convert between 
-  OpenFermion and PennyLane objects.
+* Two new functions called `qml.from_openfermion` and `qml.to_openfermion` are now available to convert between 
+  OpenFermion and PennyLane objects. This includes both fermionic and qubit operators.
   [(#5773)](https://github.com/PennyLaneAI/pennylane/pull/5773)
   [(#5808)](https://github.com/PennyLaneAI/pennylane/pull/5808)
   [(#5881)](https://github.com/PennyLaneAI/pennylane/pull/5881)
 
-  ```python
-  of_op = openfermion.FermionOperator('0^ 2')
-  pl_op = qml.from_openfermion(of_op)
-  of_op_new = qml.to_openfermion(pl_op)
+  For qubit operators:
+  ```pycon
+  >>> import openfermion
+  >>> of_qubit = 0.5 * openfermion.QubitOperator('X0 X5')
+  >>> type(of_qubit)
```

Comment:
Not a blocker, but the type information isn't _that_ useful from my perspective.